MdePkg/Library/UefiDebugLibDebugPortProtocol/DebugLib.c

```diff
@@ -280,8 +280,15 @@ DebugAssert (
     UefiDebugLibDebugPortProtocolWrite (Buffer, AsciiStrLen (Buffer));
 
     //
-    // Generate a Breakpoint, DeadLoop, or NOP based on PCD settings
+    // Generate an Assertion Break, Breakpoint, DeadLoop, or NOP based on PCD settings
     //
+    // MU_CHANGE BEGIN BREAKASSERT

```

Comment:
Technically, the MU_CHANGE should include the modification to the comment but not a big deal.

Reply:
updated